### DISCUSSION TITLE
Heroku App Down

### DISCUSSION THREAD

**Discussion by sanchivarma** [Ashutosh00710/github-readme-activity-graph#171] [Q&A] [answered]
Hi Team, 

The Heroku App seems down again. 
For URL : https://activity-graph.herokuapp.com/graph?username=<username>

Getting Error : 
`Application error
An error occurred in the application and your page could not be served. If you are the application owner, [check your logs for details](https://devcenter.heroku.com/articles/logging#view-logs). You can do this from the Heroku CLI with the command`

**Comment by Ashutosh00710** [comment, marked as answer]
Refer [this](https://github.com/Ashutosh00710/github-readme-activity-graph/issues/166#issuecomment-1336094631)
